playkit/src/main/java/com/kaltura/playkit/player/PlayerController.java

```diff
@@ -510,6 +509,14 @@ public float getPlaybackRate() {
         return Consts.PLAYBACK_SPEED_RATE_UNKNOWN;
     }
 
+    private boolean assertPlayerIstNull(String message) {
```

Comment:
I think it's better to make this function "positive". Rename to `assertPlayerNotNull`, make it return `true` if **not** null, and use it like this:

```java
if (assertPlayerNotNull("changeTrack")) {
    player.changeTrack(uniqueId);
}
```